src/dnddiceroller/Die.java

```diff
@@ -8,5 +10,30 @@
  * @author Paul Scarrone
  */
 public class Die {
-  
+  int diceSides;
+  int value;
+
+    Die(int i) {
+        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+    }
+
+    public int getDiceSides() {
+        return diceSides;
+    }
+
+    public void setDiceSides(int diceSides) {
+        this.diceSides = diceSides;
+    }
+
+    public int getValue() {
+        return value;
+    }
+
+    public void setValue(int value) {
```

Comment:
Would you want to allow someone to change the value of a roll after they rolled the die in a real game would you?